batch/test/test_batch.py

```diff
@@ -1435,6 +1436,19 @@ def test_pool_standard_instance_cheapest(client: BatchClient):
     assert 'standard' in status['status']['worker'], str((status, b.debug_info()))
 
 
+def test_gpu_accesibility_g2(client: BatchClient):
+    bb = create_batch(client)
+    resources = {'machine_type': "g2-standard-4", 'storage': '100Gi'}
+    j = bb.create_job(
+        os.environ['HAIL_GPU_IMAGE'],
+        ['python', '-c', 'import torch; assert torch.cuda.is_available() == True'],
```

Comment:
You don't need `== True` here, it means the same thing as just `is_available()`.